doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -91,12 +86,12 @@ The filter string is case sensitive. Currently, it support following filters:
 - <psu_name>.temperature: ignore temperature check for a specific PSU
 - <psu_name>.voltage: ignore voltage check for a specific PSU
 
-The default filter is to filter nothing. Unknown filters will be silently ignored. The "serivces_to_ignore" and "devices_to_ignore" section must be an string array or it will use default filter.
+The default filter is to filter nothing. Unknown filters will be silently ignored. The "services_to_ignore" and "devices_to_ignore" section must be an string array or it will use default filter.
 
 This configuration file will be platform specific and shall be added to the platform folder(/usr/share/sonic/device/{platform_name}/system_health_monitoring_config.json).
 
-#### 1.3.2 Extend the monitoring with adding user specific program to Monit
-Monit support to check program(scripts) exit status, if user want to monitor something that beyond critical serives or some special device not included in the above list, they can provide a specific scripts and add it to Monit check list, then the result can also be collected by the system health monitor. It requires 2 steps to add an external checker.
+#### 1.4.2 Extend the monitoring with adding user specific program to Monitor
+Monit support to check program(scripts) exit status, if user want to monitor something that beyond critical services or some special device not included in the above list, they can provide a specific scripts and add it to Monit check list, then the result can also be collected by the system health monitor. It requires 2 steps to add an external checker.
```

Comment:
`support` ---> `supports`?

`user want` ---> `user wants`?

`a specific scripts` ---> `specific scripts`?

`and add it to Monit check list, then` ---> `and add them to Monit checking list. Then`?

`2 steps` ---> `two steps`?